pumps/mongo_aggregate.go

```diff
@@ -49,7 +49,8 @@ type MongoAggregateConf struct {
 	// Defaults to 1000.
 	ThresholdLenTagList int `json:"threshold_len_tag_list" mapstructure:"threshold_len_tag_list"`
 	// Determines if the aggregations should be made per minute instead of per hour.
-	StoreAnalyticsPerMinute bool `json:"store_analytics_per_minute" mapstructure:"store_analytics_per_minute"`
+	// Determines the amount of time the aggregations should be made. Max (and default) = 60 - Min = 1.
+	AnalyticsStoredPerMinute int `json:"analytics_stored_per_minute" mapstructure:"analytics_stored_per_minute"`
```

Comment:
We cannot change configuration fields type - it would break backwards compatibility

Reply:
Now both configuration fields are compatible. If StoreAnalyticsPerMinute is not set (or it's false), we will look for the value of AggregationTime. If it does not exist either, set it to 60 by default.